cmake/sdformat_pc.in

```diff
@@ -1,4 +1,4 @@
-prefix="@CMAKE_INSTALL_PREFIX@"
+prefix=${pcfiledir}/../..
```

Comment:
as discussed in https://github.com/dartsim/dart/pull/1527#discussion_r525493125, this relative path is incorrect on Ubuntu

Reply:
based on a tip from @jslee02 in https://github.com/dartsim/dart/pull/1529, I've updated this in 41ac64a6adb2b719f745fffeb0859d92497ffa32 to compute the relative path to the pkgconfig folder in cmake